Source/Core/Core/IOS/ES/ES.h

```diff
@@ -190,6 +191,8 @@ class ES : public Device
   IPCCommandResult DIGetTicketView(const IOCtlVRequest& request);
   IPCCommandResult GetOwnedTitleCount(const IOCtlVRequest& request);
 
+  bool CanDeleteTitle(u64 title_id) const;
```

Comment:
This can just be a non-member internally linked function in the cpp file, since nothing is dependent on class internals within its definition.